pandas/_testing/__init__.py

```diff
@@ -309,6 +311,20 @@ def makeFloatIndex(k=10, name=None):
     return Index(values * (10 ** np.random.randint(0, 9)), name=name)
 
 
+def makeNumIndex(k=10, name=None, *, dtype):
```

Comment:
Why Hungarian case?  Shouldn't this be make_num_index?

Reply:
I guess you are just following the pattern in the other functions, despite the strangeness.

Reply:
Yes, Just following existing conventions. Updating convention would IMO fall outside of this PR.